Content.Server/GameObjects/Components/Destructible/Thresholds/Behaviors/IBehavior.cs

```diff
@@ -2,9 +2,9 @@
 using Robust.Shared.Interfaces.GameObjects;
 using Robust.Shared.Interfaces.Serialization;
 
-namespace Content.Server.GameObjects.Components.Destructible.Thresholds.Behavior
+namespace Content.Server.GameObjects.Components.Destructible.Thresholds.Behaviors
 {
-    public interface IThresholdBehavior : IExposeData
+    public interface IBehavior : IExposeData
```

Comment:
Could you please rename this again to something else? `IBehavior` alone might be real confusing given we also have `IActionBehavior, IMechanismBehavior, IItemActionBehavior`...

Reply:
The naming is something I can't quite decide on, between Thresholds, Triggers and Behaviors. I removed the Threshold prefix as they could be used for something else but given that they accept DestructibleSystem as an argument for dependencies I agree with naming it IThresholdBehavior again. Fixed.